src/pydocstyle/checker.py

```diff
@@ -44,6 +44,27 @@ def decorator(f):
     return decorator
 
 
+FSTRING_REGEX = re(r'^([rR]?)[fF]')
+
+
+def is_fstring(docstring):
+    """Return True if docstring is an f-string."""
+    return FSTRING_REGEX.match(str(docstring))
+
+
+def safe_literal_eval(string):
+    """Safely evaluate a literal even if it is an fstring."""
+    try:
+        return ast.literal_eval(string)
+    except ValueError as error:
+        # If the docstring is a fstring, it is
+        # not considered a valid docstring. See
+        # https://bugs.python.org/issue28739
+        raise ParseError(
```

Comment:
Mistakenly using an f-string for a docstring does not cause a syntax error, so we shouldn't raise a `ParseError` here.
Instead, I suggest we rename this as `eval_docstring` and just return an empty string if we find an f-string.